src/components/atoms/Link.tsx

```diff
@@ -12,12 +12,23 @@ const useStyles = makeStyles({
     },
   },
 });
+
 interface IProps extends LinkProps {
   to: string;
+  shouldReload?: boolean;
 }
 
 const Link: FC<IProps> = ({ ...props }) => {
   const classes = useStyles();
+  const { shouldReload, to } = props;
+
+  if (shouldReload) {
+    return (
+      <a className={classes.link} {...props} href={to}>
```

Comment:
Do we want it to open the link in a new window so it opens the app again?
Can we use the store for renders and reload the relevant component?

Reply:
Using an anchor tag is not correct here, and was changed to a new variant of Button. 
We want the page to reload as we can't just rerender components. There is an issue in google maps api which doesn't allow changing the language on the fly. It's described in the issue #767